packages/core/stories/avatar/avatar.stories.tsx

```diff
@@ -62,21 +62,5 @@ WithCustomSvg.args = {
 };
 
 export const ImageFallback: ComponentStory<typeof Avatar> = () => {
-  const src = "bad_url";
-  const alt = "profile";
-  const initials = "PP";
-  const [children, setChildren] = useState<ReactNode>(alt);
-
-  useEffect(() => {
-    const img = new Image();
-    img.src = src;
-    img.onerror = () => {
-      setChildren(initials);
-    };
-    img.onload = () => {
-      setChildren(<img src={src} alt={alt} />);
-    };
-  }, [src, alt]);
-
-  return <Avatar size={3}>{children}</Avatar>;
+  return <Avatar size={3} src="bad_url" name="Peter Piper" />;
```

Comment:
Since the fallback is handled within Avatar now it's simpler to show this demo

Reply:
what happens if the user wants to pass props to the image? I think this example might be redundant for an image as src, and we should have one that shows how to pass it as children when you want to add other props to your image, but it would still need to be handled outside wouldn't it?

Reply:
We should provide another example showing how img can be a child